frappe/public/js/frappe/utils/number_systems.js

```diff
@@ -31,4 +31,26 @@ export default {
 			symbol: __("K", null, "Number system"),
 		},
 	],
+	nepalese: [
+		{
+			divisor: 1.0e11,
+			symbol: __("Kh", null, "Number system"), // 10^11 is read as 1 Kharba
+		},
+		{
+			divisor: 1.0e9,
+			symbol: __("Ar", null, "Number system"), // 10^9 is read as 1 Arba
```

Comment:
This is same in indian system. Just add it there? Practically we don't see Arab in use ever, hence no one has added it